gcloud/test__helpers.py

```diff
@@ -43,6 +43,55 @@ def test_it(self):
         self.assertEqual(list(batches), [])
 
 
+class Test__UTC(unittest2.TestCase):
+
+    def _getTargetClass(self):
+        from gcloud._helpers import _UTC
+        return _UTC
+
+    def _makeOne(self):
+        return self._getTargetClass()()
+
+    def test_module_property(self):
+        from gcloud import _helpers as MUT
+
+        klass = self._getTargetClass()
+        self.assertTrue(isinstance(MUT.UTC, klass))
```

Comment:
`--recreate` your `tox` environ and it won't be an issue (at least for `tox`)

For people running just via `nosetests` it may be. I really just want to look into spoofing `sys.modules` so we can test both code paths.